_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,290 @@
+
+  - Any process or operational changes to the Core-Plans Maintenance Policies
+  - Any package feature or rewrite that breaks backwards compatibility for current users.
+  - Any Removal of a package's features (see breaking change).
+  - Changes to the standardized operational tooling in use by Core-Plans maintainers.
+  - Any changes to the standard core-plans accepted style and baseline.
+  - Any inbound Habitat change that obligates all entries in Core-Plans to change.
+
+Some changes do not require an RFC:
+
+  - Package refactors/rephrases, changes that are transparent to user.
+  - Adding net "New" software to the core repository.
+  - Additions only likely to be _noticed by_ other maintainers.
+    invisible to target package users.
+  - Bug fixes for currently available packages.
+
+If you submit a pull request to implement a new feature without going through
+the RFC process, it may be closed with a polite request to submit an RFC first.
+
+## Before creating an RFC
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other package developers
+beforehand, to ascertain that the RFC may be desirable; having a consistent
+impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking
+the idea over in the  #core-plans channel in slack as well as filing and discussing ideas on the
+[Core-Plans issue tracker](https://github.com/habitat-sh/core-plans/issues)
+
+As a rule of thumb, receiving encouraging feedback from long-standing package
+developers, and core-plans maintainers is a good indication that the RFC is 
+worth pursuing.
+
+## What the process is
+[What the process is]: #what-the-process-is
+
+In short, to get a change into Core Plans, one must first get the RFC
+merged into the RFC directory as a markdown file. At that point the RFC is
+"active" and may be implemented with the goal of eventual inclusion into Core-Plans.
+
+  - Fork the [Core Plans Repository](https://github.com/habitat-sh/core-plans)
+  - Copy `RFCs/0000-template.md` to `RFCs/0000-my-feature.md` (where "my-feature" is
+    descriptive. don't assign an RFC number yet).
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly-received.
+  - Submit a pull request. As a pull request the RFC will receive design
```

Comment:
Is this the place to mention the DCO, or rely on the DCO check to handle that feedback?